ui/ducks/app/app.ts

```diff
@@ -332,18 +333,20 @@ export default function reduceApp(
         isMouseUser: action.payload,
       };
 
-    case actionConstants.SET_SELECTED_SETTINGS_RPC_URL:
+    case actionConstants.SET_SELECTED_NETWORK_UUID:
       return {
         ...appState,
-        networksTabSelectedRpcUrl: action.payload,
+        selectedNetworkUUID: action.payload,
       };
 
-    case actionConstants.SET_NEW_NETWORK_ADDED:
+    case actionConstants.SET_NEW_NETWORK_ADDED: {
+      const { uuid, chainName } = action.payload;
```

Comment:
See note about this variable name — it might be more clear/traceable if this were `networkConfigurationUUID` / `networkConfigurationId` rather than just `uuid`.